src/controllers/userController.js

```diff
@@ -0,0 +1,11 @@
+const userService = require('../services/userService.js');
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
```

Comment:
Consider wrapping the `await userService.getAllActivated()` call in a try-catch block to handle any potential errors during the database operation. This will help prevent unhandled promise rejections and allow you to send an appropriate error response.